### DISCUSSION TITLE
How is Typst syntax related to AsciiDoc?

### DISCUSSION THREAD

**Discussion by homocomputeris** [typst/typst#268] [General]
Typst syntax reminds of AsciiDoc.

Is Typst syntax an AsciiDoc subset? Superset? Can any AsciiDoc be compiled in Typst?

**Comment by homocomputeris** [comment]
Anyone?

**Comment by lemmi** [comment]
https://typst.app/docs/reference/syntax/ looks to me quite different than asciidoc. So I don't think there is a meaningful relationship between the two.

**Comment by oceanborn2** [comment]
I think you may consider Asciidoc as a much better markdown while typst is more a replacement for Latex (precise typesetting, ...).

**Comment by sludgeFudge** [comment]
The only relation is some shorthand syntax. I'm sure you could get typst to parse through asciidoc files and generate PDFs, whether through scripting or packages, but no general support.
